conda/environment-release-build.yml

```diff
@@ -39,7 +39,7 @@ dependencies:
     # docs
     - autoclasstoc <=1.4
     - pydata_sphinx_theme ==0.9
-    - sphinx >5.1
+    - sphinx ==5.3
```

Comment:
So the problem is with sphinx 6? If so, I would use `sphinx 5.*` version specification instead, which probably wouldn't hurt to use in general, as I don't expect new major versions to work without changes. However, I suppose there is no point changing this, assuming sphinx support will be updated in the near future.